### ISSUE TITLE
Does each elements in theta range from negative infinity to positive infinity ？ 

### ISSUE THREAD

**Issue by MenSanYan** [daviddao/spatial-transformer-tensorflow#6]
Thank you for your excellent implementation of spatial transformer in Tensorflow !
However, I notice the 53-th line of "example.py" as follows:
- "h_fc1 = tf.matmul(tf.zeros([num_batch, 1200 * 1600 * 3]), W_fc1) + b_fc1".
What confuses me is  the absence of some activation function, such as tf.nn.tanh or tf.nn.sigmoid.
Without tanh or sigmoid, the range of theta is certainly not limited. What if a proper activation function is used here ? Could you please provide a good recommendation ?
Thank you !
